src/jit/gentree.h

```diff
@@ -5626,8 +5626,8 @@ struct GenCondition
         C    = Unsigned | S,    // = 14
         NC   = Unsigned | NS,   // = 15
                                 
-        FEQ  = Float | EQ,      // = 16
-        FNE  = Float | NE,      // = 17
+        FEQ  = Float | 0,       // = 16
```

Comment:
This was a mistake that slipped in with my JCC PR a while ago. Since nothing used these constants before this went unnoticed.